internal/binutils/binutils.go

```diff
@@ -179,6 +177,26 @@ func findObjdump(paths []string) (string, bool, bool) {
 	return "", false, false
 }
 
+// findExeOnDifferentPlatforms finds and returns path to preferred binary.
+// "names" is a list of names to search on both Linux and OSX.
```

Comment:
It looks unusual to quote the argument name. Haven't seen it done in function doc comments in Go programs. Even having Javadoc-style bullet list of argument names is unusual. Usually it's simply prose - e.g. see https://golang.org/pkg/io/#Copy. Let's get all comments to that common style?

Reply:
Done.